packages-next/types/src/config/index.ts

```diff
@@ -3,7 +3,7 @@ import { CorsOptions } from 'cors';
 import type { GraphQLSchema } from 'graphql';
 import type { Config } from 'apollo-server-express';
 
-import type { ImageMode, KeystoneContext } from '..';
+import type { ImageMode, FileMode, KeystoneContext } from '..';
```

Comment:
Probably a topic for discussion but ImageMode and FileMode are the same thing, does it makes sense for them to be the same thing and potentially configure the upload strategy in the same place for both? (this question is not intended to be answered in a GitHub comment now, just raising it for later)

Reply:
We think they may diverge in the future so we'll leave them separate